src/app/cluster/details/external-cluster/component.ts

```diff
@@ -180,6 +180,7 @@ export class ExternalClusterDetailsComponent implements OnInit, OnDestroy {
   disconnect(): void {
     this._clusterService.showDisconnectClusterDialog(this.cluster, this.projectID).subscribe(_ => {
       this._router.navigate(['/projects/' + this.projectID + '/clusters']);
+      this._clusterService.changeIndexClusterList(1);
```

Comment:
Having `1` as param is non-intuitive. If this will be the end solution we would have to register enum for tabs.